test/transaction.test.js

```diff
@@ -247,4 +248,15 @@ describe('transactions', function() {
         });
       });
   });
+
+  it('can return promise', function() {
```

Comment:
A suggestion (I am not confident to say it would be a better testing approach :p)

We can test a promise returned by calling those chain methods like `.then()`, see example in juggler https://github.com/strongloop/loopback-datasource-juggler/blob/de4718d5b81d29ac23597172d8ee591617924fe0/test/transaction.test.js#L94